plugin/nix-exprs/sbtix.nix

```diff
@@ -179,7 +178,7 @@ in rec {
             doDist            = true;
 
             # COURSIER_CACHE env variable is needed if one wants to use non-sbtix repositories in the below repo list, which is sometimes useful.
-            COURSIER_CACHE = "./.coursier/cache/v1";
+            COURSIER_CACHE = "~/.cache/coursier/v1"; #"./.coursier/cache/v1";
```

Comment:
This doesn't look right. `HOME` in the sandbox is set to a non-existing directory.
Usually the shell is responsible for resolving `~`, but this sets the environment variable directly, without resolving the home directory. It seems that if the builds succeeds, `~` is "resolved" to `./~/` - a relative path to a directory named literally `~`.

Reply:
good catch. I think I've copied the path from [this documentation](https://get-coursier.io/docs/cache#compatibility-with-former-versions) without translating it to the Nix syntax.